src/identity/registration.ts

```diff
@@ -11,6 +11,7 @@ export interface IdentityRegisterRequest {
   stake?: number
   beneficiary?: string
   fee?: number
+  token?: string
```

Comment:
What token? It's too generic name, I suggest `refferalCode`

Reply:
@Waldz this token, this *ReferralToken* - https://github.com/mysteriumnetwork/node/blob/master/tequilapi/contract/identity.go#L130

Reply:
I would like to have less translation, because now to change it to "referralCode" everywhere we need to make changed to node, transactor, web-ui and probably desktop app